uievents/order-of-events/mouse-events/mousemove-across-manual.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Mousemove handling across elements</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/uievents/resources/eventrecorder.js"></script>
+<style>
+#a {
+  background: red;
+  height: 120px;
+  width: 200px;
+}
+#b {
+  margin: 100px;
+  height: 120px;
+  width: 200px;
+  background: green;
+}
+#c {
+  height: 120px;
+  width: 200px;
+  background: yellow;
+}
+
+#a1 {
+  background: blue;
+  height: 120px;
+  width: 200px;
+}
+#b1 {
+  padding: 1px;
+  margin: 100px;
+  height: 120px;
+  width: 200px;
+  background: green;
+}
+#c1 {
+  height: 120px;
+  width: 200px;
+  background: black;
+}
+</style>
+<p>
+  Steps:
+  <ol>
+    <li>Move your mouse across the yellow/red <code>&lt;div&gt;</code> element quickly from right to left.
+    <li>Move your mouse across the black/blue <code>&lt;div&gt;</code> element quickly from right to left.
+    <li>If the test fails, redo it again and move faster on the black/blue <code>&lt;div&gt;</code> element next time.
```

Comment:
Why does moving faster make the test more likely to succeed? If it's just looking at order of event firing, then this should not be relevant.